### ISSUE TITLE
Exposed Dropdown Menu failing accessibility checks

### ISSUE THREAD

**Issue by rlbeckett** [material-components/material-components-android#477]
**Description:** When running instrumented tests on my app, the [accessibility checks](https://developer.android.com/reference/android/support/test/espresso/accessibility/AccessibilityChecks.html) fail now I've switched from using the default Android Spinner to the Material Design Exposed Dropdown Menu.  The output error is
```
com.google.android.apps.common.testing.accessibility.framework.integrations.AccessibilityViewCheckException: There was 1 accessibility error:
AppCompatAutoCompleteTextView{id=2131362319, res-name=dropdownSpinner, visibility=VISIBLE, width=954, height=153, 
has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=true, is-enabled=true, is-focused=false, 
is-focusable=true, is-layout-requested=false, is-selected=false, layout-params=android.widget.FrameLayout$LayoutParams@5bdbf4, 
tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, text=2nd, input-type=0, ime-target=false, 
has-links=false}: View is missing speakable text needed for a screen reader
```
 I'm using databinding within the xml to setup the input but it behaves perfectly fine when manually tested and talkback does (probably) give enough information to use the input.  

There's also no obvious difference between how I'm implementing the Exposed Dropdown Menu and how I've implemented the TextInputEditText - they both provide the hint to the TextInputLayout.  However when observing the Espresso output, the hint has been passed to the TextInputEditText but not the AutoCompleteTextView.

**Expected behavior:** Accessibility tests should pass.

**Source code:**
```
<com.google.android.material.textfield.TextInputLayout
        android:id="@+id/dropdownSpinnerLayout"
        style="@style/Widget.AppTheme.DropdownLayout"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_marginTop="@dimen/small_margin"
        android:layout_marginBottom="@dimen/small_margin"
        android:hint="@{bindingModel.inputHint}"
        android:labelFor="@id/dropdownSpinner">

        <AutoCompleteTextView
            android:id="@+id/dropdownSpinner"
            style="@style/Widget.AppTheme.DropdownSpinner"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:editable="false"
            android:completionThreshold="@{999999}"
            android:onTextChanged="@{(chars, start, begin, count) -> bindingModel.transformLabelToValue(chars.toString())}"
            android:text="@{bindingModel.label}"
            app:dropDownItemLayout="@{@layout/app_dropdownitem_layout}"
            app:dropDownItems="@{bindingModel.items}" />

    </com.google.android.material.textfield.TextInputLayout>
```

**Android API version:** Android API version 28 mostly but varies between devices

**Material Library version:** Material Android Library version: 1.1.0-alpha08

**Device:** Both real devices (One Plus 6T, Samsung S7, and others) as well as emulated devices

**Mitigation** I tried adding the hint to the AutoCompleteTextView but, whilst it fixed the accessibility test failure, it changed the hint behaviour so it wasn't aligned with the other app components. For now I'm able to target the accessibility checks at everything bar the new dropdown in order to run my test suites but that isn't a long term solution.

**Comment by leticiarossi** [comment]
Fixed in https://github.com/material-components/material-components-android/commit/d2870fbb3fd170712abfb6d6ff8bccf799360f5b
